documentation/manual-lobster_cpptest.md

```diff
@@ -0,0 +1,115 @@
+# Tracing to C/C++ test requirements
+
+## Setup and requirements
+
+You will need a C/C++ file or a directory containing.
+
+C/C++ files should have one of these extensions to be evaluated by this tool: `.cpp, .cc, .c or .h`
```

Comment:
Side remark: That's true for now. In the future the file ending will be configurable.